pkg/podmounter/mppod/path.go

```diff
@@ -0,0 +1,27 @@
+package mppod
+
+import (
+	"path/filepath"
+)
+
+// KnownPathMountSock is the path of Unix socket thats going to be used during exchange of mount options
+// between Mountpoint Pod and the CSI Driver Node Pod.
+const KnownPathMountSock = "mount.sock"
+
+// CommunicationDirName is the name of `emptyDir` volume each Mountpoint Pod will create
+// for the communication between Mountpoint Pod and the CSI Driver Node Pod.
+// Each Pod will have a different view for the files inside this folder,
```

Comment:
'view' isn't the right word here - we want to say that the directories are different between MP pods

Reply:
The "view" part was to refer different Pods will have different paths to the same objects, thus different views. But I'm open to any suggestions